src/Slider/Thumbs.tsx

```diff
@@ -0,0 +1,69 @@
+import { useRef } from "react";
+import { VisuallyHidden, mergeProps, useFocusRing, useSliderThumb } from "react-aria";
+
+const getRailWidth = (lowerInputThumbProps, higherInputThumbProps) => parseFloat(higherInputThumbProps.style?.left.toString().slice(0, -1)) - parseFloat(lowerInputThumbProps.style?.left.toString().slice(0, -1));
+
+const useThumb = (index, trackRef, name, state) => {
+  const inputRef = useRef(null);
+  const { thumbProps, inputProps } = useSliderThumb({
+    index,
+    trackRef,
+    inputRef,
+    name
+  }, state);
+  const { focusProps } = useFocusRing();
```

Comment:
This looks keyboard navigable (nice!) but we should visually indicate focus somehow.

It would be good to sync with @hillarypollak on this to see if there's a good way to present a focus/active indicator for the currently focused slider handle.

My proposal would be a transparent theme primary ring around the handle dot while it's active.

Reply:
agreed on this